### PR TITLE
Start screen UI challenge/yefim yakunin

### PR DESCRIPTION
**Pull request by fimillion**
Done Start Screen challenge

### CODE REVIEW

**Comment by fimillion** [comment]
Help pleas ))

**Comment by alexnikol** [comment]
Wrong button height, button font weight (should be medium - not regular)

**Comment by alexnikol** [comment]
You need to add 3 images (not just 1)
Screenshot 2023-02-25 at 19 30 38
Screenshot 2023-02-25 at 19 30 13

File: UIMentoringSession/MenteesChallenges/Start Screen UI Challenge/YefimYakunin/YefimYakunin_StartScreenViewController.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import UIKit
+
+class YefimYakunin_StartScreenViewController: UIViewController {
```
Comment (alexnikol) [inline UIMentoringSession/MenteesChallenges/Start Screen UI Challenge/YefimYakunin/YefimYakunin_StartScreenViewController.swift:3]: Make it final

File: UIMentoringSession/MenteesChallenges/Start Screen UI Challenge/YefimYakunin/YefimYakunin_StartScreenViewController.swift [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
```
Comment (alexnikol) [inline UIMentoringSession/MenteesChallenges/Start Screen UI Challenge/YefimYakunin/YefimYakunin_StartScreenViewController.swift:4]: Remove space between class statement and variables

File: UIMentoringSession/MenteesChallenges/Start Screen UI Challenge/YefimYakunin/YefimYakunin_StartScreenViewController.swift [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    }
+    
+    init() {
+        super.init(nibName: "YefimYakunin_StartScreenViewController", bundle: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
```
Comment (alexnikol) [inline UIMentoringSession/MenteesChallenges/Start Screen UI Challenge/YefimYakunin/YefimYakunin_StartScreenViewController.swift:20]: Remove space before closing "}"

**Comment by alexnikol** [comment]
Set all texts in the controller (on viewDidLoad). Don't leave texts in xibs and storyboards

**Review by alexnikol — APPROVED** [review]

**Pull request merged**